offline.js

```diff
@@ -497,3 +546,46 @@ const processFileData = function(ref, data) {
     Sefaria.cacheVersionsAvailableBySection(data.sectionRef, data.versions);
     return data;
 };
+
+/**
+* Checks if a book's index JSON fil©index JSON exists or the unzip was successful, false otherwise
+*/
+async function ensureTitleUnzipped(title) {
+
+    const titleIsOffline = hasOfflineTitle(title);
+
+    if (!titleIsOffline) {
+        return false; // Title Doesn't exist offline
+    } else {
+        const indexJsonPath = _JSONSourcePath(title + "_index"); // Path like /path/to/Title_index.json
+        const indexJsonExists = await fileExists(indexJsonPath);
+
+        if (indexJsonExists) {
+            return true; // Already unzipped
+        }
+
+        // Index JSON doesn't exist, check for the zip file
```

Comment:
Zip file has to exist based on hasOfflineTitle === true (as noted below). This makes the function unnecessarily complicated.